flash/core/model.py

```diff
@@ -115,10 +115,12 @@ def predict(
             The post-processed model predictions
 
         """
-        data_pipeline = data_pipeline or self.data_pipeline
-        batch = x if skip_collate_fn else data_pipeline.collate_fn(x)
-        batch_x, batch_y = batch if len(batch) == 2 else (batch, None)
-        predictions = self.forward(batch_x)
+        self.eval()
+        with torch.no_grad():
```

Comment:
maybe you can set it as methods decorator...

Reply:
Done !